src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin: 40px 16px 0;
```

Comment:
The top margin for `&__title` seems inconsistent with the given design. Verify that the margin-top is set according to the design requirements.